docs/resources/Webhook.md

```diff
@@ -188,20 +188,20 @@ Refer to [Uploading Files](#DOCS_REFERENCE/uploading-files) for details on attac
+| tts               | boolean                                                                              | true if this is a TTS message                                                                                                                                                                             | false                        |
+| embeds            | array of up to 10 [embed](#DOCS_RESOURCES_CHANNEL/embed-object) objects              | embedded `rich` content                                                                                                                                                                                   | one of content, file, embeds |
+| allowed_mentions  | [allowed mention object](#DOCS_RESOURCES_CHANNEL/allowed-mentions-object)            | allowed mentions for the message                                                                                                                                                                          | false                        |
+| components \*     | array of [message component](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/component-object) | the components to include with the message                                                                                                                                                                | false                        |
+| files[n] \*\*     | file contents                                                                        | the contents of the file being sent                                                                                                                                                                       | one of content, file, embeds |
+| payload_json \*\* | string                                                                               | JSON encoded body of non-file params                                                                                                                                                                      | `multipart/form-data` only   |
+| attachments \*\*  | array of partial [attachment](#DOCS_RESOURCES_CHANNEL/attachment-object) objects     | attachment objects with filename and description                                                                                                                                                          | false                        |
+| flags             | integer                                                                              | [message flags](#DOCS_RESOURCES_CHANNEL/message-object-message-flags) combined as a [bitfield](https://en.wikipedia.org/wiki/Bit_field) (only `SUPPRESS_EMBEDS`, and `SUPPRESS_NOTIFICATIONS` can be set) | false                        |
```

Comment:
this comma is wrong, as Oxford comma only exists in series of three or more things.

Reply:
> this comma is wrong, as Oxford comma only exists in series of three or more things.

This is why GitHub needs a "🤓" reaction.